docs/metrics.md

```diff
@@ -164,6 +168,45 @@ Open a web browser to [http://127.0.0.1:3000](http://127.0.0.1:3000), you should
 
 > Makefile targets `make grafana-portforward`,`make kind-grafana-portforward` and `make minikube-grafana-portforward`.
 
+### Stackdriver installation
+
+In order to use [Stackdriver monitoring](https://app.google.stackdriver.com) you should enable Stackdriver Monitoring API on Google Cloud Console. You need to grant all necessary permissions to the users (see [Access Control Guide](https://cloud.google.com/monitoring/access-control)). Stackdriver exporter uses a strategy called Application Default Credentials (ADC) to find your application's credentials. Details could be found here [Setting Up Authentication for Server to Server Production Applications](https://cloud.google.com/docs/authentication/production).
+
+Default metrics exporter is Prometheus. In order to change it to Stackdriver update following helm parameters in [values file](../install/helm/agones/values.yaml):
+```
+agones:
+  metrics:
+    prometheusEnabled: false
+    prometheusServiceDiscovery: false
+    stackdriverEnabled: true
+    stackdriverProjectID: ""
+```
+
+With this configuration only Stackdriver exporter would be used instead of Prometheus exporter.
+
+Run the following command to install Agones to a cluster with an updated configuration:
+```
+make install
```

Comment:
Let's not put make commands in out user documentation if we can avoid it. Let's have explicit commands n our documentation instead.